notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    latexify(width_scale_factor=2, fig_height=2)
</pre><p>width_scale_factor shoulde be 3 since we have 3 figures.</p><p>fig_height should be 1.5 since  I have checked in latex source code</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>